scripts/west_commands/boards.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import collections
+import os
+import re
+import shutil
+import subprocess
+import sys
+import textwrap
+
+from west import log
+from west import cmake
+from west.configuration import config
+from west.commands import WestCommand
+from cmake import run_cmake
+
+BOARDS_DESCRIPTION = '''\
+Display information about boards
+'''
+
+class Boards(WestCommand):
+
+    def __init__(self):
+        super(Boards, self).__init__(
+            'boards',
+            # Keep this in sync with the string in west-commands.yml.
+            'display information about supported boards',
+            BOARDS_DESCRIPTION,
+            accepts_unknown_args=False)
+
+    def do_add_parser(self, parser_adder):
+        default_fmt = '{name:32} ({arch})'
```

Comment:
Just to keep things simple, I think this should be `'{name}'`.

Reply:
sure, no objections